examples/v1alpha2/hyperband-example.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   namespace: kubeflow
   name: hyperband-example
 spec:
+  parallelTrialCount: 9
```

Comment:
may be added maxTrials too?

Reply:
hyperband algorithm can terminate itself when loop exceeds. we doesn't need set maxTrials for it.